app/javascript/widget/components/TeamAvailability.vue

```diff
@@ -82,7 +83,9 @@ export default {
         return this.replyTimeStatus;
       }
       if (workingHoursEnabled) {
-        return this.outOfOfficeMessage;
+        return this.isOnline
+          ? this.replyTimeStatus
+          : `We will be back online ${this.timeLeftToBackInOnline}`;
```

Comment:
Same as above.

Reply:
Can't this be in the mixing as it is repeated in 2 different components?

Reply:
Done https://github.com/chatwoot/chatwoot/pull/6902/commits/2c0e179d2a22769ac7797a325de475ab2b416ce7